hiplot/test_experiment.py

```diff
@@ -39,6 +40,24 @@ def test_from_dataframe() -> None:
     xp.validate()
     xp._asdict()
 
+def test_from_optuna() -> None:
+
+    def objective(trial):
+        x = trial.suggest_float("x", -1, 1)
+        return x ** 2
```

Comment:
Missing some typing here - let's not worry too much about tests typing tho, this should work
```suggestion
    def objective(trial: Any) -> float:
        x = trial.suggest_float("x", -1, 1)
        return x ** 2
```